Raeng Castagna/week_2/Animation/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+ <head>
+  <meta charset="utf-8" />
+  <title>Cat Walk</title>
+  </script>
+ </head>
+ <body>
+
+  <img style="position:absolute;" src="http://www.anniemation.com/clip_art/images/cat-walk.gif">
```

Comment:
Try and avoid using inline styles. Better to have them in a separate CSS file - or if in this situation because we're changing so much in JS have this in your JS file.